projects/plugins/boost/app/assets/src/js/features/ReactComponent.svelte

```diff
@@ -7,11 +7,13 @@
 	let root;
 	export let inline = false;
 	afterUpdate( () => {
-		// Remove `inline` from the props passed to the component.
-		// eslint-disable-next-line @typescript-eslint/no-unused-vars, no-shadow
-		const { this: component, children, inline, ...otherProps } = $$props;
-		root = WPElement.createRoot( container );
-		root.render( React.createElement( component, otherProps, children ) );
+		const { this: component, children, inline, ...props } = $$props;
```

Comment:
I may have messed inline while trying to merge trunk. Please take a look at that as well.

Reply:
Thanks for raising this. It looks like the only change you undid through the merge was @pyronaur renamed `props` to `otherProps` for clarity. I've re-established that.

However, the linter picked up that `inline` inside `afterUpdate` is shadowing `inline` from the parent context, so I have cleaned that up too by destructuring `inline` into `inlineProp` and copying it to the `inline` variable in code.

Either way, this is temp code. So long as it doesn't cause problems during the remainder of the migration we'll be ok.

32485c45c3

Reply:
Thank you